contract/src/instantiate.rs

```diff
@@ -12,5 +12,7 @@ pub fn instantiate(
 
     cw2::set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
+    sanity(deps.storage, &env);
```

Comment:
Should this be only done on `#[cfg(debug_assertions)]` ?

Reply:
Yes and no. Yes: it should only run on debug_assertions. No: because I'm lazy, I set up the `sanity` function itself to simply be a no-op if debug_assertions are disabled. That way I don't have to worry about that code accidentally shipping to production.